### PR TITLE
chore: add deprecation notice for openebs/chart helm registry

### PR DESCRIPTION
**Pull request by niladrih**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,22 @@
...
+## ⚠️ Deprecation Notice: Helm Chart Registry Migration
+
+The Helm chart registry at "https://openebs.github.io/charts" is deprecated and will be moved to a new location soon.
+
+**Old Registry URL:** https://openebs.github.io/charts  
+**New Registry URL:** https://openebs.github.io/openebs  
+
+To ensure seamless access to OpenEBS Helm charts, please update your configurations to use the new registry URL.
+
+Additionally, the older registry will be relocated to the 'github.com/openebs-archive' GitHub organization shortly.
```
Comment (avishnu) [inline README.md:12]: replace shortly with after SIX months

File: index.md [hunk 2]
```diff
@@ -1,5 +1,22 @@
...
+## ⚠️ Deprecation Notice: Helm Chart Registry Migration
+
+The Helm chart registry at "https://openebs.github.io/charts" is deprecated and will be moved to a new location soon.
+
+**Old Registry URL:** https://openebs.github.io/charts  
+**New Registry URL:** https://openebs.github.io/openebs  
+
+To ensure seamless access to OpenEBS Helm charts, please update your configurations to use the new registry URL.
+
+Additionally, the older registry will be relocated to the 'github.com/openebs-archive' GitHub organization shortly.
```
Comment (avishnu) [inline index.md:12]: Same comment as above

File: README.md (hunk 1 above)
Comment (niladrih) [reply README.md]: Added relocation deadline.

File: index.md (hunk 2 above)
Comment (niladrih) [reply index.md]: Added relocation deadline.

**Review by avishnu — APPROVED** [review]

**Pull request merged**